Doc/library/stdtypes.rst

```diff
@@ -3826,9 +3826,13 @@ another set.  The :class:`frozenset` type is immutable and :term:`hashable` ---
 its contents cannot be altered after it is created; it can therefore be used as
 a dictionary key or as an element of another set.
 
-Non-empty sets (not frozensets) can be created by placing a comma-separated list
-of elements within braces, for example: ``{'jack', 'sjoerd'}``, in addition to the
-:class:`set` constructor.
+Sets can be created by several ways:
+
+* Using a comma-separated list of elements within braces: ``{'jack', 'sjoerd'}``
+* Using :class:`set` constructor: ``set()``, ``set('foobar')``,
+  ``set(['a', 'b', 'foo'])``
+* Using a set comprehension: ``{x for x in ()}``,
+  ``{c for c in 'abracadabra' if c not in 'abc'}``
```

Comment:
I move this into class dict, not sure if you are saying this section. But it is now consistent with `list` and `tuple`.